### PR TITLE
Feature/OL43-add_todo

### PR DESCRIPTION
**Pull request by CavalcanteLucas**
**Link to task:** 
https://labcodes.atlassian.net/browse/OL-43

**How to test:**
- With a user logged in having one or more TodoLists, click on the "+" (plus) button in the TodoListDetail page, check if modal with form pops up;
- Fill the form and check if Todo is inserted at the bottom of the listing of Todos.

**Description of your solution:**
This PR implements the API endpoint for enabling CREATE requests of a Todo instance. This PR also creates the interface to action that fetches the CREATE endpoint and handle state updates / redirect action for the REQUEST / SUCCESS / FAILURE cases. Tests ensure that creating Todos require Token authorization, and that it is a private method among different users. A "+" (plus) button in the botton of TodoListDetail page represents the "add Todo" button. When this button is selected, a modal with a form is open. After proper filling, an instance of Todo is created and included in the current page.

### CODE REVIEW

File: labdale/todo_lists/tests/test_todo_lists.py [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
-User = get_user_model()
+from labdale.todo_lists.models import TodoList
+from labdale.todo_lists.serializers import TodoListSerializer
 
 
 class TodoListTests(TestCase):
+    def test_todo_list__str__(self):
+        todo_list = baker.make("TodoList")
+        self.assertEqual(todo_list.__str__(), todo_list.title)
+        self.assertEqual(str(todo_list), todo_list.title)
```
Comment (cacoze) [inline labdale/todo_lists/tests/test_todo_lists.py:17]: hey @CavalcanteLucas  those lines are pretty much the same. Can you remove one?

File: labdale/todo_lists/tests/test_todos.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+from labdale.todo_lists.models import Todo, TodoList
+from labdale.todo_lists.serializers import TodoSerializer
+
+
+class TodoTests(TestCase):
+    def test_todo__str__(self):
+        todo = baker.make("Todo")
+        self.assertEqual(todo.__str__(), todo.title)
+        self.assertEqual(str(todo), todo.title)
```
Comment (cacoze) [inline labdale/todo_lists/tests/test_todos.py:17]: same here @CavalcanteLucas

File: labdale/todo_lists/tests/test_todo_lists.py (hunk 1 above)
Comment (CavalcanteLucas) [reply labdale/todo_lists/tests/test_todo_lists.py:17]: Thank you for noting. Fixed.

File: labdale/todo_lists/tests/test_todos.py (hunk 2 above)
Comment (CavalcanteLucas) [reply labdale/todo_lists/tests/test_todos.py:17]: Fixed.

**Pull request merged**